CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CDTDatastore CHANGELOG
 
-## Unreleased
+## 2.0.0 (2018-02-14)
```

Comment:
Preferable to group the same entry types together, I guess this all just got merged in random orders.

Reply:
See 8fc467e